web/packages/teleport/src/Discover/useDiscover.tsx

```diff
@@ -553,6 +553,22 @@ export type NodeMeta = BaseMeta & {
   node: Node;
 };
 
+export type DatabaseServiceDeploy = {
+  /**
+   * serviceDeployedMethod flag will be undefined if user skipped
+   * deploying service (service already existed).
+   */
```

Comment:
Is this comment added in the correct place? It seems to me that only `serviceDeploy` can be undefined, not the `method` here?